buildall/README.md

```diff
@@ -18,13 +23,23 @@ Linux/Unix like environment:
   - The built docker images require around 5GB of space
   - Approximately roughly 50GB for build cache. See `DOCKER_PRUNE` below
     to optionally prune the build cache during the build process.
-- Helm 3
+- Helm 3 (3.17 or later recommended)
 - Python 3 (in-support version such as 3.10 or later, tested on 3.12)
-- Mage
-
-Additionally, some of the containers that are build require the following
-
-- Open Policy Agent v0.69.0 (or possibly later) for infra-core repo
+- Mage 1.15 or later
+
+Additionally, some of the Makefiles in component repos require the following
+dependencies to run properly:
+
+- Open Policy Agent (`opa` cli tool) v0.69.0 (or possibly later) for infra-core repo
+- golangci-lint (v1.64.8 or later)
+- go-junit-report (v2.1.0 or later)
+- oapi-codegen (v1.12.0 or later)
+- protoc-gen-doc (1.5.1 or later)
+- buf (1.45.0 or later)
+- protoc-gen-go-grpc (1.2.0 or later)
+- protoc-gen-go (v1.30.0 or later)
```

Comment:
`protoc-gen-openapi` (made available through https://github.com/google/gnostic/tree/main/cmd/protoc-gen-openapi)  and `protoc-gen-grpc-gateway-v2.26.3` (from https://github.com/grpc-ecosystem/grpc-gateway/releases/download/v2.26.3/protoc-gen-grpc-gateway-v2.26.3-linux-x86_64) plugins are also needed by protoc.

Reply:
I added these (with specific versions used in infra-core repo) in [29d4280](https://github.com/open-edge-platform/edge-manageability-framework/pull/247/commits/29d4280042d43ed3e6789f8f53c33021eb848857)